06_gpu_and_ml/dreambooth_app.py

```diff
@@ -42,20 +42,26 @@
 stub = modal.Stub(name="example-dreambooth-app")
 
 # Commit in `diffusers` to checkout `train_dreambooth.py` from.
-DREAMBOOTH_SCRIPT_COMMIT_HASH = "2868d99181976753b10fa6a4bb0695982f463cbe"
-
-image = modal.Image.debian_slim().pip_install(
-    "accelerate",
-    "smart_open",
-    # Do not update without updating the commit hash above.
-    "diffusers[torch]~=0.11.1",
-    "ftfy",
-    "transformers",
-    "torch",
-    "torchvision",
-    "triton",
-    "xformers==0.0.16rc393",
-    "gradio~=3.10",
+GIT_SHA = "ed616bd8a8740927770eebe017aedb6204c6105f"
+
+image = (
+    modal.Image.debian_slim()
+    .pip_install(
+        "accelerate",
+        "datasets",
+        "ftfy",
+        "gradio~=3.10",
+        "smart_open",
```

Comment:
Do we still need this?

Reply:
it's still used for downloading images, but i'll change that in a subsequent PR